package.json

```diff
@@ -1,54 +1,29 @@
 {
   "name": "hubot",
   "version": "2.19.0",
```

Comment:
Assuming this is something that should never get published to `npm` (seems likely), we should set `private: true` here